test/e2e/storage/csi_mock/csi_volume_expansion.go

```diff
@@ -388,7 +393,7 @@ var _ = utils.SIGDescribe("CSI Mock volume expansion", func() {
 		}
 	})
 
-	ginkgo.Context("Expansion with recovery[Feature:RecoverVolumeExpansionFailure]", func() {
+	ginkgo.Context("Expansion with recovery", func() {
```

Comment:
Wrong feature tag. It should be the one on L204: "[Feature:CSINodeExpandSecret]"